src/components/OverviewCard.jsx

```diff
@@ -42,28 +176,124 @@ const OverviewCard = ({
+    setCanShowFileMoveDropdown(!canShowFileMoveDropdown)
+    setCanShowDropdown(!canShowDropdown)
+  }
+  
   return (
+    <>
     <Link className={`${contentStyles.component} ${contentStyles.card} ${elementStyles.card}`} to={generateLink()}>
       <div id={itemIndex} className={contentStyles.componentInfo}>
         <div className={contentStyles.componentCategory}>{category ? category : ''}</div>
         <h1 className={contentStyles.componentTitle}>{generateTitle()}</h1>
         <p className={contentStyles.componentDate}>{date ? date : ''}</p>
       </div>
       {settingsToggle &&
-        <div className={contentStyles.componentIcon}>
-          <button
+        <div className="position-relative mt-auto">
+          <button 
             type="button"
             id={`settings-${itemIndex}`}
             onClick={(e) => {
-              e.preventDefault(); 
-              settingsToggle(e)}}
-            className={contentStyles.componentIcon}
+                e.stopPropagation();
+                e.preventDefault();
+                setCanShowDropdown(true)
+              }}
+            className={`${contentStyles.componentIcon}`}
           >
-            <i id={`settingsIcon-${itemIndex}`} className="bx bx-cog" />
+            <i id={`settingsIcon-${itemIndex}`} className="bx bx-dots-vertical-rounded" />
           </button>
+          {canShowDropdown &&
+            <div className={`position-absolute ${elementStyles.dropdown}`} ref={dropdownRef} tabIndex={2} onBlur={()=>setCanShowDropdown(false)}>
+              <MenuItem handler={settingsToggle} id={`settings-${itemIndex}`}>
+                <i className="bx bx-sm bx-edit"/>
+                Edit details
+              </MenuItem>
+              <MenuItem handler={toggleDropdownModals}>
+                <i className="bx bx-sm bx-folder"/>
+                Move to
+                <i className="bx bx-sm bx-chevron-right ml-auto"/>
+              </MenuItem>
+              <MenuItem handler={() => setCanShowDeleteWarningModal(true)}>
+                <i className="bx bx-sm bx-trash text-danger"/>
+                Delete item
+              </MenuItem>
+          </div>}
+          {canShowFileMoveDropdown &&
+            <div className={`position-absolute ${elementStyles.dropdown}`} ref={fileMoveDropdownRef} tabIndex={1} onBlur={handleBlur}>
+              <MenuItem className={`d-flex`}>
+                <i className="bx bx-sm bx-arrow-back" onClick={toggleDropdownModals}/>
+                Move to
+              </MenuItem>
+              <hr/>
+              {category && !isResource &&
+                <MenuItem handler={() => moveFile('')}>
+                  Unlinked Page
+                </MenuItem>
+              }
+              {allCategories
+                ?
+                allCategories.map((categoryName) => {
+                  if (categoryName !== category) {
+                    return (
+                      <MenuItem handler={() => moveFile(categoryName)}>
```

Comment:
We should have a `key` for each `MenuItem`

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/219/commits/64d412c454f06e845802ac10d78b3b950d0cd1fd